x/marker/types/immutable_addresses.go

```diff
@@ -0,0 +1,48 @@
+package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+// ImmutableAccAddresses holds some AccAddresses in a way that they cannot be changed after creation.
+type ImmutableAccAddresses struct {
```

Comment:
A bit of overkill as it adds a bit of code that fees like it should be useful as part of the base SDK directly into a module ... but still a worthwhile change. (feel free to dismiss)

Reply:
Yeah... I mostly just wanted to see what it'd look like, but kind of liked it.

It has the nice benefit of turning `IsReqAttrBypassAddr` from O(7) into O(1), which probably actually doesn't even matter in the long run, but oh well.